### PR TITLE
ci(pie-monorepo): DSW-1422 add node 20 support

### PR DESCRIPTION
**Pull request by siggerzz**
## Author Checklist (complete before requesting a review)
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -319,10 +304,8 @@ jobs:
     needs:
     - lint-styles
     - lint-js
-    - build-ubuntu-node-16
-    - build-windows-node-16
-    - build-ubuntu-node-18
-    - build-windows-node-18
+    - build-windows-node-20
```
Comment (ashleynolan) [inline .github/workflows/ci.yml:307]: Do we think there's value in testing in node 18 as well, or are we happy just testing in node 20 now?

Comment (siggerzz) [reply .github/workflows/ci.yml:307]: After speaking with @xander-marjoram and @jamieomaguire, I think we'll look to move these other builds to PIE Aperture, to ensure our components work with older versions of Node 👍🏼

Comment (ashleynolan) [reply .github/workflows/ci.yml:307]: Happy to do this for Node 16. I think the danger with doing that for 18 too is that we're likely one of the first teams to update our stack to use that, so I think we should keep the node 18 checks it for now until we have checks elsewhere that cover that gap

File: .husky/commit-msg [hunk 2]
```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-#Check commit message format
-npx --no -- commitlint --edit ${1}
+
+# Check if it's a merge commit, if so, skip commitlint
+if ! git rev-parse --verify MERGE_HEAD >/dev/null 2>&1; then
```
Comment (siggerzz) [inline .husky/commit-msg:6]: This was bugging me so I wanted to address it 😁

**Review by raoufswe — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**